tools/build_updated_packages.rb

```diff
@@ -65,7 +65,7 @@ def self.check_build_uploads(architectures_to_check = nil, name = nil)
         unless @remote_hash == binary_sha256_hash[arch.to_sym]
           puts "#{arch}/#{name}: Adjusting sha256sum in package file to the remote binary sha256sum".lightpurple
           puts "#{binary_sha256_hash[arch.to_sym]} =>\n#{@remote_hash}".blue
-          system "sed -i 's,#{binary_sha256_hash[arch.to_sym]},#{@remote_hash},g' packages/#{name}.rb"
+          system "sed 's,#{binary_sha256_hash[arch.to_sym]},#{@remote_hash},g;w packages/#{name}.rb.new' packages/#{name}.rb && mv packages/#{name}.rb.new packages/#{name}.rb"
```

Comment:
No need to create `.rb.new`.

Reply:
This is because sed's in place replace is throwing an error. I'm not sure if just writing to the same file works, but I'm willing to try it.